documentation/operators/src/faq/mixnodes-faq.md

```diff
@@ -1,22 +1,37 @@
 # Frequently Asked Questions
 
-## Nym Nodes
+## Nym Mixnet
+
+To see different stats about Nym Mixnet live, we recommend you to visit [status.notrustverify.ch](https://status.notrustverify.ch/d/CW3L7dVVk/nym-mixnet?orgId=1) built by [No Trust Verify](https://notrustverify.ch/) crew, one of the squads within Nym core community.
+
+<iframe src="https://status.notrustverify.ch/d-solo/CW3L7dVVk/nym-mixnet?orgId=1&from=1702215592419&to=1704807592419&panelId=12" width="800" height="400" frameborder="0"></iframe>
+
+
+### Is there an explorer for Nym Mixnet?
+
+Yes, there are several places, some are built by Nym core community:
+
+* [Nym Explorer](https://explorer.nymtech.net/)
+* [Guru Explorer](https://mixnet.explorers.guru/)
+* [ExploreNYM](https://explorenym.net/)
 
 ### What determines the rewards when running a Mix Node?
 
 The stake required for a Mix Node to achieve maximum rewards is called Mix Node saturation point. This is calculated from the staking supply (all circulating supply + part of unlocked tokens). The target level of staking is to have 50% of the staking supply locked in Mix Nodes.
 
-The node stake saturation point, which we denote by Nsat, is given by the stake supply, target level of staking divided by the number of rewarded (active) nodes. 
+The node stake saturation point, which we denote by Nsat, is given by the stake supply, target level of staking divided by the number of rewarded (active) nodes.
```

Comment:
active mixnode != rewarded mixnode.

we have following sets of mixnodes:
- bonded => self explanatory, essentially all of them
- rewarded => nodes that were selected to be in the rewarded set, i.e. will receive **some** rewards for this epoch (assuming non-zero performance), this can be further subdivided into:
  - active => top N nodes of the rewarded set (currently all of them, but doesn't have to be), nodes that will actually be used by the clients and iirc will also get better rewards
  - standby => bottom N nodes of the rewarded set - won't be used for mixing apart from testing
- inactive => nodes that haven't been selected to be in the rewarded set